lib/internal/errors.js

```diff
@@ -518,6 +518,10 @@ function getMessage(key, args) {
  * @returns {Error}
  */
 function uvException(ctx) {
+  console.log(errmap);
+  console.log(ctx);
+  console.log(ctx.errno);
+  console.log(errmap.get);
```

Comment:
are those `console.log` expected?

Reply:
Yes, everything from the second commit is going to be removed (including this). I was debugging an issue with a test on FreeBSD.